tests/unit/LegalObligation.spec.ts

```diff
@@ -23,12 +23,12 @@ const newIncorporationFiling = [
   {
     availableOnPaperOnly: false,
     displayName: 'Incorporation Application',
-    effectiveDate: new Date('2019-06-02T19:22:59.003777+00:00'),
```

Comment:
This is wrong since effectiveDate should be a string, not a Date object.

Ditto for submittedDate below.